iOS/DuckDuckGo/SettingsSubscriptionView.swift

```diff
@@ -264,28 +264,28 @@ struct SettingsSubscriptionView: View {
 
         if subscriptionFeatures.contains(.dataBrokerProtection) {
             let hasDBPEntitlement = userEntitlements.contains(.dataBrokerProtection)
-
-            if DataBrokerProtectionIOSManager.isDBPStaticallyEnabled {
-                NavigationLink(destination: LazyView(DataBrokerProtectionViewControllerRepresentation(dbpViewControllerProvider: DataBrokerProtectionIOSManager.shared!)), isActive: $isShowingDBP) {
-                    SettingsCellView(
-                        label: UserText.settingsPProDBPTitle,
-                        image: Image(uiImage: DesignSystemImages.Color.Size24.identity),
-                        statusIndicator: StatusIndicatorView(status: hasDBPEntitlement ? .on : .off),
-                        isGreyedOut: !hasDBPEntitlement
-                    )
-                }
-                .disabled(!hasDBPEntitlement)
-            } else {
-                NavigationLink(destination: LazyView(SubscriptionPIRMoveToDesktopView()), isActive: $isShowingDBP) {
-                    SettingsCellView(
-                        label: UserText.settingsPProDBPTitle,
-                        image: Image(uiImage: DesignSystemImages.Color.Size24.identity),
-                        statusIndicator: StatusIndicatorView(status: hasDBPEntitlement ? .on : .off),
-                        isGreyedOut: !hasDBPEntitlement
-                    )
+            let hasValidStoredProfile = settingsViewModel.dataBrokerProtectionIOSManager
+                .flatMap { try? $0.meetsProfileRunPrequisite } ?? false
+            let statusIndicator: StatusIndicator = hasDBPEntitlement && hasValidStoredProfile ? .on : .off
```

Comment:
One last tweak to request then I think we're good to go here: today in production, the PIR badge is hardcoded to `.on`, but after this change it will appear as `.off`. Would we be able to fall back to `.on` as a default in the case where we don't pass the `isDBPStaticallyEnabled` check? Thanks!